azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -227,23 +232,20 @@ public AzkabanWebServer(Server server, Props props) throws Exception {
     }
 
     configureMBeanServer();
-    if (props.getBoolean(ServerProperties.IS_METRICS_ENABLED, false)) {
-      startWebMetrics();
-    }
   }
 
-  private void startWebMetrics() throws Exception{
-    MetricRegistry metrics = MetricsManager.INSTANCE.getRegistry();
-    MetricsWebRegister execWorker = new MetricsWebRegister.MetricsWebRegisterBuilder("WEB")
-        .addExecutorManager(getExecutorManager())
-        .build();
-    execWorker.addExecutorManagerMetrics(metrics);
+  private void startWebMetrics() throws Exception {
+    WebMetrics.INSTANCE.addExecutorManagerMetrics(getExecutorManager());
 
+    MetricRegistry registry = MetricsManager.INSTANCE.getRegistry();
+    MetricsUtility.addGauge("JETTY-NumIdleThreads", registry, queuedThreadPool::getIdleThreads);
+    MetricsUtility.addGauge("JETTY-NumTotalThreads", registry, queuedThreadPool::getThreads);
+    MetricsUtility.addGauge("JETTY-NumQueuedThreads", registry, queuedThreadPool::getQueueSize);
```

Comment:
This is not the number of threads. It is the queue size. The metric name doesn't look right.